examples/java/spring-boot/src/main/java/uk/gov/api/springboot/domain/services/package-info.java

```diff
@@ -0,0 +1,4 @@
+@DomainServiceRing
```

Comment:
I am ignoring the Sonar finding here intentionally, as we will implement Domain Services before long